Subject: [PATCH] Small try to product
---
Index: base/settings.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/base/settings.py b/base/settings.py
--- a/base/settings.py	(revision 9c43501587c64eba34efb930ecb1dc65ea74b734)
+++ b/base/settings.py	(revision a9ddc26666fe04fb29a4437c16f4279b2aa9ce75)
@@ -133,10 +133,7 @@
     'DEFAULT_RENDERER_CLASSES': [
         'rest_framework.renderers.JSONRenderer',
         'rest_framework.renderers.BrowsableAPIRenderer',
-    ],
-    'DEFAULT_AUTHENTICATION_CLASSES': (
-        'rest_framework_simplejwt.authentication.JWTAuthentication',
-    )
+    ]
 }
 
 SIMPLE_JWT = {
